core/handler/device/store.go

```diff
@@ -39,18 +41,22 @@ func NewRedisDeviceStore(client *redis.Client, prefix string) *RedisDeviceStore
 	for v, f := range migrate.DeviceMigrations(prefix) {
 		store.AddMigration(v, f)
 	}
+	s, _ := client.Get(prefix + ":" + redisDevicePrefix + ":whitelist").Result()
+	l := strings.Split(s, ":")
```

Comment:
Whitelist is too generic imo. It should be clear that it's a filter for attribute keys that are published upstream